components/cf-app-ssh/backend/cf_websocket_app_ssh.go

```diff
@@ -96,19 +96,14 @@ func (cfAppSsh *CFAppSsh) appSSH(c echo.Context) error {
 	// cf:APP-GUID/APP-INSTANCE-INDEX@SSH-ENDPOINT
 	username := fmt.Sprintf("cf:%s/%s@%s", appGUID, appInstance, host)
 
-	clientID, err := p.GetClientId("cf")
-	if err != nil {
-		return echo.NewHTTPError(http.StatusInternalServerError, "Could not get client ID forCloud Foundry")
-	}
-
 	// Need to get SSH Code
 	// Refresh token first - makes sure it will be valid when we make the request to get the code
-	refreshedTokenRec, err := p.RefreshToken(cnsiRecord.SkipSSLValidation, cnsiRecord.GUID, userGUID, clientID, "", cnsiRecord.TokenEndpoint)
+	refreshedTokenRec, err := p.RefreshToken(cnsiRecord.SkipSSLValidation, cnsiRecord.GUID, userGUID, p.GetConfig().ConsoleConfig.ConsoleClient, p.GetConfig().ConsoleConfig.ConsoleClientSecret, cnsiRecord.TokenEndpoint)
```

Comment:
The console client and secret can be different to the one used for CF - I think this should be reverted - I believe you have addressed configured per-endpoint client and client secret in another PR.

Reply:
yes I accidentally pushed this change